packages/react-client/src/ReactFlightClient.js

```diff
@@ -584,8 +584,8 @@ export function parseModelString(
           case BLOCKED:
             const parentChunk = initializingChunk;
             chunk.then(
-              createModelResolver(parentChunk, parentObject, key),
-              createModelReject(parentChunk),
+              createValueResolver(parentChunk, parentObject, key),
+              createValueReject(parentChunk),
```

Comment:
I think it should be either `createValueResolver`/`createValueRejecter` or `createValueResolve`/`createValueReject`.